notNeededPackages.json

```diff
@@ -3186,6 +3186,12 @@
             "sourceRepoURL": "https://github.com/sindresorhus/rev-hash",
             "asOfVersion": "3.0.0"
         },
+        {
+            "libraryName": "rfc4648",
+            "typingsPackageName": "rfc4648",
+            "sourceRepoURL": "https://github.com/swansontec/rfc4648",
```

Comment:
URL is not valid. I believe it should be `https://github.com/swansontec/rfc4648.js`